src/components/sidebar/right-sidebar/Filters.tsx

```diff
@@ -86,9 +95,29 @@ export default function Filters({ page }: FilterProps) {
       page)
   }
 
+  const clearFilter = () => {
+    // remove selected options in dropdown (frontend)
+    if (unityAlignedOnlyItem.current.querySelector(".clear")) unityAlignedOnlyItem.current.querySelector(".clear").click();
+    if (sourceTypeItem.current.querySelector(".clear")) sourceTypeItem.current.querySelector(".clear").click();
+    if (overallRiskOfBiasItem.current.querySelector(".clear")) overallRiskOfBiasItem.current.querySelector(".clear").click();
+    if (populationGroupItem.current.querySelector(".clear")) populationGroupItem.current.querySelector(".clear").click();
+    if (sexItem.current.querySelector(".clear")) sexItem.current.querySelector(".clear").click();
+    if (ageItem.current.querySelector(".clear")) ageItem.current.querySelector(".clear").click();
+    if (testTypeItem.current.querySelector(".clear")) testTypeItem.current.querySelector(".clear").click();
+    if (isotypesReportedItem.current.querySelector(".clear")) isotypesReportedItem.current.querySelector(".clear").click();
+    if (antibodyTargetItem.current.querySelector(".clear")) antibodyTargetItem.current.querySelector(".clear").click();
+
+      // remove filters in backend

```

Comment:
As discussed on basecamp this should also clear filters on frontend, should remove the block above in order to prevent unnecessary api calls

Reply:
Thanks to your absolute genius, we are able to clean up all of this logic :D